sbpy/photometry/core.py

```diff
@@ -848,7 +957,13 @@ def _hgphi(pha, i):
 
     @staticmethod
     def evaluate(pha, hh, gg):
-        return hh-2.5*np.log10((1-gg)*HG._hgphi(pha, 1)+gg*HG._hgphi(pha, 2))
+        func = (1-gg)*HG._hgphi(pha, 1)+gg*HG._hgphi(pha, 2)
```

Comment:
docstring

Reply:
Do we really need docstrings for all the `.evaluate` and `.fit_deriv` method?  For people who care about defining their own model classes, they would understand what these are; for those who just need to use the model classes we provided here, they never need to directly call these methods.